pkg/cmd/kafka/update/update.go

```diff
@@ -153,15 +169,40 @@ func run(opts *options) error {
 		if err != nil {
 			return err
 		}
-	}
 
-	if opts.owner == kafkaInstance.GetOwner() {
-		opts.logger.Info(opts.localizer.MustLocalize("kafka.update.log.info.sameOwnerNoChanges", localize.NewEntry("Owner", opts.owner), localize.NewEntry("InstanceName", kafkaInstance.GetName())))
-		return nil
+		reauthenticationPrompt := &survey.Select{
+			Message: opts.localizer.MustLocalize("kafka.update.input.message.reauthentication"),
+			Options: []string{"true", "false"},
+			Default: strconv.FormatBool(kafkaInstance.GetReauthenticationEnabled()),
+		}
+
+		err = survey.AskOne(reauthenticationPrompt, &opts.reauth)
+		if err != nil {
+			return err
+		}
 	}
 
 	updateObj := kafkamgmtclient.NewKafkaUpdateRequest()
-	updateObj.SetOwner(opts.owner)
+
+	if opts.owner != "" && opts.owner != kafkaInstance.GetOwner() {
```

Comment:
```suggestion
	if opts.owner != ""  {
	  if opts.owner == kafkaInstance.GetOwner() {
	    // warn user
	  }
```

Reply:
Not very much for of it as the existing update flows don't check for same values.

Reply:
Not sure what you mean?